File: src/main/java/com/gmail/borlandlp/minigamesdtools/activepoints/BuildSchema.java
package com.gmail.borlandlp.minigamesdtools.activepoints;

import org.bukkit.Location;
import org.bukkit.Material;

import java.util.HashMap;
import java.util.Map;

public class BuildSchema {
    private Map<Location, Material> blocks = new HashMap<>();

    public BuildSchema(Map<Location, Material> map) {
        this.blocks = map;
    }

    public Map<Location, Material> getSchema() {
        return blocks;
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/activepoints/BuildSchema.kt
package com.gmail.borlandlp.minigamesdtools.activepoints

import org.bukkit.Location
import org.bukkit.Material
import java.util.*

class BuildSchema(map: Map<Location, Material>) {
    var schema: Map<Location, Material> = mutableMapOf()

    init {
        schema = map
    }
}